src/Resources/assets/js/main.js

```diff
@@ -1,4 +1,4 @@
-import onboardingWizard from './onboardingWizard';
+import onboardingWizard from './OnboardingWizard';
```

Comment:
for semantics, please change to 
`import OnboardingWizard from './OnboardingWizard';`
and 3rd line:
`const tour = new OnboardingWizard();`
script should still work properly after that